src/components/Header.js

```diff
@@ -20,24 +20,45 @@ const Header = () => {
         <NavLink to="/index" className="nav-link">
           Available Homesteads
         </NavLink>
-        <NavLink to="/signin" className="nav-link">
-          Sign In
-        </NavLink>
-        <Button
-          tag={Link}
-          to="/signup"
-          className="nav-link"
-          style={{
-            borderRadius: "20px",
-            padding: "10px 20px",
-            backgroundColor: "#CBA86B",
-            color: "black",
-            border: "none",
-            marginRight: "2rem",
-          }}
-        >
-          Sign Up
-        </Button>
+        {!currentUser && (
+          <NavLink to="/signin" className="nav-link">
+            Sign In
+          </NavLink>
+        )}
+        {!currentUser && (
+          <Button
+            tag={Link}
+            to="/signup"
+            className="nav-link"
+            style={{
+              borderRadius: "20px",
+              padding: "10px 20px",
+              backgroundColor: "#CBA86B",
+              color: "black",
+              border: "none",
+            }}
```

Comment:
I know we talked about this button with the Link tag before, and I am pretty sure this won't work. If it does, great! But I would double check.

Reply:
when we look at the application tab/ local storage it deletes the user token. Even with that link all the buttons work.